src/Utilities/Compiler/Options/EditorConfigParser.cs

```diff
@@ -70,15 +70,15 @@ public static CategorizedAnalyzerConfigOptions Parse(SourceText text)
 
                     Debug.Assert(!string.IsNullOrEmpty(key));
                     Debug.Assert(key == key.Trim());
-                    Debug.Assert(value == value?.Trim());
+                    Debug.Assert(value == value.Trim());
```

Comment:
Actually, I am not, but I cannot think of a test where it will be null. I thought better to remove this unnecessary defensive check, especially given this code path is only taken on user-opt in (if they add some editorconfig option). We will hopefully get a bug report and can fix it with regression test.